docs/src/modules/operations/pages/services/deploy-service.adoc

```diff
@@ -171,16 +171,24 @@ For further details, see xref:operations:projects/external-container-registries.
 [#apply]
 == Using service descriptors
 
-Akka services can also be described and managed with *YAML service descriptors*.
+Akka services can also be described and managed with *YAML service descriptors*. See xref:reference:descriptors/service-descriptor.adoc[].
+
+Once you have <<_deploying_a_service, deployed your service>>, you can modify it by editing its service descriptor:
```

Comment:
I think we should move this to a separate `===` section, after "Redeploying with a descriptor".

Edit would be a very rare operation in my opinion, unless possibly when toying around. Instead, the normal cycle is to edit the descriptor in git, then apply it again.

Reply:
OK. Let's do that.

Reply:
done, now.